### PR TITLE
Add unit conversions

### PR DESCRIPTION
**Pull request by herbertludowieg**
Added unit conversion code from exa.util.conversions and the constants from exa.util.constants.

Fixed a small bug in the adf/output.py

Added a column for the gaussian Fchk parser that we need for some other things.

### CODE REVIEW

**Comment by herbertludowieg** [comment]
Still have to fix up the tests for the traits and other things.

File: exatomic/adf/output.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def parse_basis_set_order(self):
                         data['n'].append(n)
                         data['r'].append(r)
         data['set'] = data.pop('seht')
-        data['frame'] = 0
+        data['frame'] = [0]*len(data['set'])
```
Comment (avmarchenko) [inline exatomic/adf/output.py:188]: Can you explain this change? Doing `data["frame"] = 0` sets the entire series "frame" to 0. The change does the same exact thing that pandas does automatically.

File: exatomic/gaussian/output.py [hunk 2]
```diff
@@ -499,11 +507,13 @@ def parse_frequency(self):
...
             freqs = ln[freq_start:].split()
             # get the ir intensities
-            ir_int = self[lno+3][freq_start:].split()
+            ir_int = list(map(float, self[lno+3][freq_start:].split()))
+            r_mass = list(map(float, self[lno+1][freq_start:].split()))
             nfreqs = len(freqs)
             # Get just the atom displacement vectors
             start = lno + span_freq_vals
             stop = start + span
+            #print(lno, start, stop, self[start], self[stop])
```
Comment (avmarchenko) [inline exatomic/gaussian/output.py:516]: I would delete debug print statements. Likewise throughout.

File: exatomic/gaussian/output.py [hunk 3]
```diff
@@ -835,7 +847,7 @@ def parse_frequency_ext(self):
         if not found:
             return
         else:
-            found = self.find(_renmode, _redisp, _refinfo, _remass, keys_only=True)
+            found = self.find(_renmode, _refinfo, keys_only=True)
```
Comment (avmarchenko) [inline exatomic/gaussian/output.py:850]: Is this a backwards incompatible change or was this previously unused?

File: exatomic/interfaces/xyz.py [hunk 4]
```diff
@@ -45,19 +45,39 @@ def parse_atom(self, unit='Angstrom', names=('symbol', 'x', 'y', 'z')):
...
-        # The following algorithm works for both trajectory files and single xyz files
-        nats = pd.Series(df[df[['y', 'z']].isnull().all(axis=1)].index)
-        nats = nats[nats.diff() != 1].values
-        comments = nats + 1
-        nats = df.loc[nats, 'symbol']
-        comments = df.loc[comments, :].dropna(how='all').index
-        initials = nats.index.values.astype(np.int64) + 2
-        counts = nats.values.astype(np.int64)
-        frame, _, indices = starts_counts(initials, counts)
-        df = df[df.index.isin(indices)]
```
Comment (avmarchenko) [inline exatomic/interfaces/xyz.py:60]: Can you speak to why changes to `xyz.py` are needed? Because there are no changes to the tests, it seems like the previous implementation was working correctly. If there is a gap that the above changes address, we should capture that by updating the tests to cover that edge case.

File: exatomic/adf/output.py (hunk 1 above)
Comment (herbertludowieg) [reply exatomic/adf/output.py:188]: I remember at the time I was having an issue because it initializes everything as a dictionary from collections and it wasn't doing that automatically. A couple lines down it converts the dictionary into a pandas dataframe.

File: exatomic/interfaces/xyz.py (hunk 4 above)
Comment (herbertludowieg) [reply exatomic/interfaces/xyz.py:60]: Yeah, so there was an issue when the comment line had more than four strings separated by a space. I noticed it when it runs through the test with `'ZrO2.xyz'` where the comment line has 5 space delimited strings.

Something I looked into was using the `on_bad_lines='skip'` and we could just have the code look in the `df.loc[0,'symbol']` value for the number of atoms. I'm not sure how to then find the first non-comment line. For most of them, it works. Just that one case it does not.

I'm open to any suggestions you have on how to improve that section of the code. Mind you this is only an issue when using pandas>=1.4.0.

File: exatomic/gaussian/output.py (hunk 3 above)
Comment (herbertludowieg) [reply exatomic/gaussian/output.py:850]: They are unused. The `_remass` is for the isotopic masses used. The `_redisp` is for the normal modes which is not what we are looking at in this method.

File: exatomic/gaussian/output.py (hunk 2 above)
Comment (herbertludowieg) [reply exatomic/gaussian/output.py]: Made a new commit addressing this.

File: exatomic/gaussian/output.py [hunk 5]
```diff
@@ -481,8 +481,16 @@ def parse_frequency(self):
         # get the location of the Atom labels in the frequency blocks
         # the line below this is where all of the normal mode displacement data begins
         found_atom = np.array(self.regex('Atom', start=start_read, stop=stop_read, keys_only=True)) + start_read
-        # Total lines per block minus the unnecessary ones
-        span = starts[1] - found_atom[0] - 3
+        try:
+            # Total lines per block minus the unnecessary ones
+            span = starts[1] - found_atom[0] - 3
+        except IndexError:
```
[the remark this answers is not in the document]
Comment (herbertludowieg) [reply exatomic/gaussian/output.py:487]: By starts. I am pretty sure that this was brought about when there were less than three frequencies which only generated one "Frequencies" label when using HPModes. Updated test with the edge case to come.

**Review by avmarchenko — APPROVED** [review]

**Comment by avmarchenko** [comment]
Approved. CI isn't fully working; let me know and I will merge.

File: exatomic/gaussian/output.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (herbertludowieg) [reply exatomic/gaussian/output.py:487]: Updated the unit test for the edge case in a commit.

**Comment by herbertludowieg** [comment]
Will do, thank you!

**Review by avmarchenko — APPROVED** [review]
